install/deb/templates/web/skel/document_errors/403.html

```diff
@@ -13,7 +13,7 @@
           "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji",
           "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
         text-shadow: 0px 1px 1px rgba(255, 255, 255, 0.75);
-        text-align: center !important;
+        text-align: center;
```

Comment:
We should "consolidate" all in /common/templates/web/skel/ folder instead of each Debian / RHEL

Reply:
39e8c3e